### DISCUSSION TITLE
LEMON F405 Firmware

### DISCUSSION THREAD

**Discussion by cmsurveys** [iNavFlight/inav#8260] [Q&A] [answered]
Can anybody advise where I can get the latest firmware for the LEMON F405?

**Comment by JulioCesarMatias** [comment, marked as answer]
iNav does not support this flight controller

**Reply by cmsurveys** [reply]
Hi Julio,  the board has INAVFLIGHT F405 printed on it but it is advertised as LEMON F405.
 Can you advise what I can do with this board as I cannot connect to anything with it at the moment.

**Comment by stronnag** [comment]
* If the board already has firmware loaded by the vendor, use the CLI `version` command to see what firmware is loaded.
* If it does not have firmware loaded, ask the vendor which target you should use.

The INAV developers have NO knowledge of this board.

**Comment by DzikuVx** [comment]
Just like @stronnag and @JulioCesarMatias said: we have no idea what this board is, nobody consulted it with us or even informed about it's existence. 
Bottom line: for any support ask vendor/seller/manufacturer

**Comment by jpfree** [comment]
# version
# INAV/LEMONF405 1.9.0 Apr  3 2018 / 01:03:47 ()

WHICH FIRMWARE COULD I FLASH?

**Reply by erstec** [reply]
https://github.com/iNavFlight/inav/discussions/8260#discussioncomment-3219722
